libsplinter/src/admin/service/shared.rs

```diff
@@ -1500,12 +1795,6 @@ impl AdminServiceShared {
                     return Err(AdminSharedError::ValidationFailed(
                         "Proposed circuit cannot have a display name on protocol 1".to_string(),
                     ));
-                } else if circuit.get_circuit_status()
```

Comment:
why was this deleted?

Reply:
This is checking the protocol version, rather than the `circuit_version` (This was an error I was initially running into while testing the disband request). I added a check to verify the circuit version in the `validate_disband_request` rather than the admin service protocol, as it is doing here.